client/object_delete.go

```diff
@@ -25,42 +24,13 @@ var (
 
 // PrmObjectDelete groups optional parameters of ObjectDelete operation.
 type PrmObjectDelete struct {
+	sessionContainer
 	meta v2session.RequestMetaHeader
```

Comment:
This creates a split brain situation with two metas.

Reply:
This one is obviously excess. Removed